public/main.css

```diff
@@ -28,4 +28,15 @@ h1 {
     background-color: #000000;
     color:#ff3333;
     font-weight:bold;
+}
+
+.todo-complete {
+    color:green;
+    font-weight:bold;
+}
+
+#count-label {
+    margin-top:20px;
+    font-size:150%;
+    font-weight:bold;
 }
```

Comment:
I hate when other people point this out, but I guess I'm going to.... Add a new line at the end of your file here. Apparently github flags it as there is a chance that something doesn't deal with it properly.